### ISSUE TITLE
Falcoctl install uses an older version of DaemonSet

### ISSUE THREAD

**Issue by kris-nova** [falcosecurity/falcoctl#72]
**What happened**:

Running a Kubernetes 1.17 cluster trying to install the daemonset with version `extension/v1beta` fails as this API expects `apps/v1`

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Comment by kris-nova** [comment]
This brings to mind a broader effort to support a helm chart as part of the effort here

**Comment by rashmi43** [comment]
I would like to take this up.

**Comment by KyleMartin901** [comment]
Hi @rashmi43 this is how I was able to reproduce the error hope that it helps.

I created a v1.17.3 [KIND Kubernetes](https://github.com/kubernetes-sigs/kind) cluster

```
kind create cluster --image kindest/node:v1.17.2
```

Once I had that running I downloaded the latest released version of falcoctl v0.0.7

```
GO111MODULE="on" go get github.com/falcosecurity/falcoctl@v0.0.7
```

I then install Falco using the falcoctl Kubernetes installer

```
falcoctl install falco
2020-02-21T22:39:34+11:00 [✿]  Falco namespace created: falco-ns
2020-02-21T22:39:34+11:00 [✿]  Falco service accounts created: falco-ns
2020-02-21T22:39:34+11:00 [✿]  Falco config map created: falco-ns
2020-02-21T22:39:34+11:00 [✖]  Error creating daemon set: the server could not find the requested resource
2020-02-21T22:39:34+11:00 [✖]  Fatal error: the server could not find the requested resource
```

As you can see the it will error when trying to create the daemon set. 

If you look at the v0.0.7 code you will find the creation of the DaemonSet on [line 127](https://github.com/falcosecurity/falcoctl/blob/v0.0.7/pkg/kubernetes/falco.go#L127). However if you look in master you will see that this has now been fixed by the commit https://github.com/falcosecurity/falcoctl/commit/a5e1b4a651e08c6c362263944cd07763eedd1273 where the API version was changed from `apps/v1beta2` to `apps/v1`. You can see that the error was caused by the fact that the deamonset resource no longer supports apps/v1beta2 api requests from Kubernetes v1.16 onwards https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/  
>DaemonSet in the extensions/v1beta1 and apps/v1beta2 API versions is no longer served

Hope that helps see how you would reproduce it and see how you would of fixed it sorry someone had already fixed it so didn't get a chance to have a go at it.

**Comment by sureshpalemoni** [comment]
@KyleMartin901  How do I install falco on mac?

**Comment by rashmi43** [comment]
Thanks @KyleMartin901 .

**Comment by rashmi43** [comment]
Can I work on the helm chart creation @kris-nova ?

**Comment by leogr** [comment]
We already have a stable helm chart here: https://github.com/helm/charts/tree/master/stable/falco

**Comment by leogr** [comment]
Update: we should take care of this when working on #86

**Comment by leogr** [comment]
Closed in favor of #95
